src/launcher.rs

```diff
@@ -0,0 +1,93 @@
+use clap::Arg;
+use clap_nested::{Command, Commander, MultiCommand};
+
+use std::{env, fs};
+use std::path::Path;
+
+use crate::config::get_tools_path;
+
+fn get_windows_terminal_fragments_path(title: &str) -> String {
+    let local_app_data = env::var("LocalAppData").unwrap();
+    format!("{}/Microsoft/Windows Terminal/Fragments/{}", local_app_data, title)
+}
+
+fn get_add_runner(_args: &str, matches: &clap::ArgMatches<'_>) -> std::result::Result<(), clap::Error> {
+    let title = matches.value_of("title").unwrap();
+    let idf_path = matches.value_of("idf-path").unwrap();
+    let fragments_path = get_windows_terminal_fragments_path(title);
+    let tools_path = get_tools_path();
+
+    // After fresh installation of Windows Terminal the fragment path does not exist.
+    // Microsoft recommends to create one
+    // if !Path::new(&fragments_path).exists() {
+    fs::create_dir_all(&fragments_path)?;
+    let fragment_json_path = format!("{}/fragment.json", fragments_path);
+    println!("Updating Windows Terminal Fragment: {}", fragment_json_path);
+
+    let command_line = format!("C:/WINDOWS/System32/WindowsPowerShell/v1.0/powershell.exe -ExecutionPolicy Bypass -NoExit -File {}/Initialize-Idf.ps1", tools_path);

```

Comment:
I would suggest to use `windir` env variable instead of hardcoding path on drive C:

Reply:
Great idea. Thank you. Updated.